homeassistant/components/sensor/bart.py

```diff
@@ -0,0 +1,256 @@
+"""
+Support for the BART (Bay Area Rapid Transit) API.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/sensor.bart/
+"""
+import logging
+from datetime import timedelta
+
+from homeassistant.helpers.entity import Entity
+from homeassistant.util import Throttle
+
+_LOGGER = logging.getLogger(__name__)
+
+ICON = 'mdi:train'
+
+# Return cached results if last scan was less then this time ago.
+MIN_TIME_BETWEEN_UPDATES = timedelta(seconds=60)
+
+
+def parse_api_response(raw_xml):
+    """Parse the BART API response."""
+    import xml.etree.ElementTree as etree
+    if isinstance(raw_xml, bytes):
```

Comment:
I believe this was because of how urlopen worked. The check is unneeded now with `requests`.